### ISSUE TITLE
Wrong login history displayed for user

### ISSUE THREAD

**Issue by listerr** [inex/IXP-Manager#594]
##### ISSUE TYPE

Bug Report

##### VERSION

 ```
define( 'APPLICATION_VERSION', '5.1.0' );
define( 'APPLICATION_VERDATE', '2019072300' );

```

##### SUMMARY
From the user table, clicking on the login history displays the history for another user.

##### STEPS TO REPRODUCE
Go to user table in either user/admin view
Click display login history / history button
Login history for a different user appears.

Example, for user with id 880, it loads the history for user 722.

I think the link (https://portal.lonap.net/login-history/view/722)  is meant to refer to the id in the link table and then display all matching login history for this user instance, and not the user table?

```
mysql> SELECT * FROM customer_to_users WHERE user_id = 880;
+-----+-------------+---------+-------+--------------------------------------------------+---------------------+-----------------+---------------------+
| id  | customer_id | user_id | privs | extra_attributes                                 | last_login_date     | last_login_from | created_at          |
+-----+-------------+---------+-------+--------------------------------------------------+---------------------+-----------------+---------------------+
| 722 |         307 |     880 |     2 | {"created_by": {"type": "user", "user_id": 880}} | 2019-11-13 10:13:50 | x.x.x.x  | 2019-10-25 15:42:50 |
+-----+-------------+---------+-------+--------------------------------------------------+---------------------+-----------------+---------------------+
1 row in set (0.00 sec)
```

**Comment by barryo** [comment]
@yannrobin  - @listerr says this is still broken in inex/IXP-Manager#595 - can you check please?

**Comment by barryo** [comment]
@yannrobin has confirmed fixed in v5.5.0
